google/cloud/pubsublite/internal/alarm_registry.h

```diff
@@ -41,6 +42,8 @@ class AlarmRegistry {
   AlarmRegistry& operator=(AlarmRegistry&&) = delete;
   // When CancelToken is destroyed, the alarm will not be running and will never
   // run again.
+  // All `CancelToken`s returned from `RegisterAlarm` must be destroyed before
```

Comment:
This is a bad constraint to need- remove it

Reply:
You also don't appear to need this anywhere

Reply:
oh yea I think this was leftover from when I had `period` and `on_alarm` in a member variable vector